ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml

```diff
@@ -38,18 +38,18 @@ tests:
-          {"step": "install", "failure_type": "pod_failure", "classification": "OpenTelemetry operator install", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "install", "failure_type": "test_failure", "classification": "OpenTelemetry operator install", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "distributed-tracing-install-*", "failure_type": "pod_failure", "classification": "Distributed tracing operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "install", "failure_type": "pod_failure", "classification": "OpenTelemetry operator install", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "install", "failure_type": "test_failure", "classification": "OpenTelemetry operator install", "jira_project": "TRACING", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "distributed-tracing-install-*", "failure_type": "pod_failure", "classification": "Distributed tracing operators", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
```

Comment:
Does anyone know why we created unique test refs for this scenario
https://steps.ci.openshift.org/reference/operatorhub-subscribe-amq-streams
https://steps.ci.openshift.org/reference/operatorhub-subscribe-elasticsearch-operator

This could have been installed using the install-operators ref for simplicity, but there may be a reason this was needed that I'm unaware of.

If there isn't a reason then we can file a ticket in the backlog to fix it in another PR, no need to hold this PR for this issue as it's not really related to the work for this PR.

Reply:
After team discussion, the switch to using `install-operator` will be handled as a PL within our team